src/pageServices/contentScripts/highlight.js

```diff
@@ -190,7 +195,7 @@ export const highlightOnPage = () => {
     div.className = getClassName(predictedElement);
     div.setAttribute("jdn-highlight", true);
     div.setAttribute("jdn-status", predictedElement.locator.taskStatus);
-    const labelContainer = document.createElement("div");
```

Comment:
what problem does it solves?

Reply:
getting the id from evt.target and reducing extra wrapper